.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
Is there a way to verify the wrapper JAR? Do they publish SHAs of it?

Reply:
I don't think there is a way. Should the wrapper be removed?

Reply:
I guess that's a philosophical question :)

Reply:
Perhaps this could help: https://maven.apache.org/plugins/maven-wrapper-plugin/index.html

Reply:
See the "source" type here: https://maven.apache.org/plugins/maven-wrapper-plugin/usage.html This would avoid comitting any opaque binary file, IIUC.

Reply:
Yes, that's one of the new features of Maven 3.7.0^h^h 4.0

I believe this get's in the way

```The version of the wrapper, default value is the Runtime version of Maven, should be at least 3.7.0
User property is: wrapperVersion.
```

Given that wrapper scripts for `3.6.3` do not exist I get the following error with type `script` and `source`. Type 1bin` would be the same as what we have right now.

```$ gm org.apache.maven.plugins:maven-wrapper-plugin:3.0.1:wrapper
Using maven at '/Users/aalmiray/.sdkman/candidates/maven/current/bin/mvn' to run buildFile '/Users/aalmiray/dev/github/jfrunit/pom.xml':
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 1 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 1 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-wrapper-plugin/3.0.1/maven-wrapper-plugin-3.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-wrapper-plugin/3.0.1/maven-wrapper-plugin-3.0.1.pom (4.8 kB at 8.2 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-wrapper-plugin/3.0.1/maven-wrapper-plugin-3.0.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-wrapper-plugin/3.0.1/maven-wrapper-plugin-3.0.1.jar (24 kB at 515 kB/s)
[INFO] 
[INFO] --------------------< org.moditect.jfrunit:jfrunit >--------------------
[INFO] Building JfrUnit 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-wrapper-plugin:3.0.1:wrapper (default-cli) @ jfrunit ---
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven-wrapper/3.6.3/apache-maven-wrapper-3.6.3.pom
[WARNING] The POM for org.apache.maven:apache-maven-wrapper:zip:script:3.6.3 is missing, no dependency information available
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven-wrapper/3.6.3/apache-maven-wrapper-3.6.3-script.zip
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.123 s
[INFO] Finished at: 2021-02-20T22:43:11+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-wrapper-plugin:3.0.1:wrapper (default-cli) on project jfrunit: Could not find artifact org.apache.maven:apache-maven-wrapper:zip:script:3.6.3 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

Reply:
Do we need 3.6.3 though?

Reply:
That's the latest. 3.7.0 was renamed to 4.0 which has yet to be released.